.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_solo
+
+platforms:
+  - name: ubuntu-12.04
+  - name: centos-6.4
+
+suites:
+  - name: default
+    run_list:
+      - recipe[sonar::default]
```

Comment:
Also I could recommend you to try to use kitchen-docker, which could really be much faster then vagrant, but I am not really sure how stable it is.